custom_components/mail_and_packages/helpers.py

```diff
@@ -217,6 +217,12 @@ def process_emails(hass: HomeAssistant, config: ConfigEntry) -> dict:
         except Exception as err:
             _LOGGER.error("Error updating sensor: %s reason: %s", sensor, err)
 
+    # Update usps_mail_delivered
+    try:
+        fetch(hass, config, account, data, "usps_mail_delivered")
```

Comment:
The sensor would already be in the `resources` and searched for twice.

Reply:
I added some additional logging in my testing and I don't think it's in `resources`.  If `resources` is set by `get_resources()`, `SENSOR_TYPES` does not contain the BinarySensors. I think.

Reply:
Please correct me in I am wrong.  This is my first time digging into HA. Does `config.get(CONF_RESOURCES)` get set by:
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/4a397286698f3b3ea4d26542b279071b25716284/custom_components/mail_and_packages/config_flow.py#L211-L213
If so `get_resources()` here:
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/4a397286698f3b3ea4d26542b279071b25716284/custom_components/mail_and_packages/helpers.py#L94-L103
uses `SENSOR_TYPES` defined here:
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/4a397286698f3b3ea4d26542b279071b25716284/custom_components/mail_and_packages/const.py#L651
which does not include `usps_mail_delivered` defined here:
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/4a397286698f3b3ea4d26542b279071b25716284/custom_components/mail_and_packages/const.py#L1157

Reply:
Ah you are correct, it's been excluded from the resources in error. That's where the bug needs to be fixed.

Reply:
That was my guess. I wasn't exactly sure how to do that.

Reply:
In addition to the Binary sensors, I think the Images sensors are missing too.

Reply:
I put in a PR on your fork that would solve the issue.

Reply:
> In addition to the Binary sensors, I think the Images sensors might be missing too, as in they are always created and not selected during setup.

This is by design (for now).

Reply:
> I put in a PR on your fork that would solve the issue.

I'll pull it in and give it a try.

Reply:
Feel free to give it a run on a test HA instance if you like too, all the sensors will now be sorted alphabetically.

Reply:
I've finally started to test it now.  Running into a weird situation where the Logbook and History show the sensor turning on, but the sensor still shows off. 
![image](https://github.com/user-attachments/assets/086a5793-c8d2-408d-81e3-6be279838bf9)

Reply:
if you refresh does it show "on"?
if so, this will be an bug in HA not updating the front end.

Reply:
Interesting.  Yep. Issue with HA front end. I had clicked around to different views in HA and even restarted the Mail and Packages integration, but it wasn't until I refreshed the browser that it updated.

Reply:
I've noticed this issue after the 2024.8.2 update.